packages/client/lib/rpc/modules/txpool.ts

```diff
@@ -0,0 +1,43 @@
+import { middleware } from '../validation'
+
+import type { EthereumClient } from '../..'
+import type { FullEthereumService } from '../../service'
+import type { TxPool as Pool } from '../../service/txpool'
+import type { VM } from '@ethereumjs/vm'
+
+/**
+ * web3_* RPC module
+ * @memberof module:rpc/modules
+ */
+export class TxPool {
+  private _txpool: Pool
+  private _vm: VM
+  /**
+   * Create web3_* RPC module
+   * @param client Client to which the module binds
+   */
+  constructor(client: EthereumClient) {
+    const service = client.services.find((s) => s.name === 'eth') as FullEthereumService
+    this._txpool = service.txPool
+    this._vm = service.execution.vm
+    this.content = middleware(this.content.bind(this), 0, [])
+  }
+
+  /**
+   * Returns the contents of the transaction pool
+   * @param params An empty array
+   */
+  content(_params = []) {
+    const pending = new Map()
+    for (const pool of this._txpool.pool) {
+      const pendingForAcct = new Map<bigint, any>()
+      for (const tx of pool[1]) {
+        pendingForAcct.set(tx.tx.nonce, tx.tx.toJSON())
```

Comment:
Can't we just `pool[1].map((e) => e.tx.toJSON()`?

Reply:
Yes, yes we can.  :-)  Good catch.